<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>871</width>
      <height>631</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="MainPage">
     <widget class="QGroupBox" name="grpIsolatedPlaces">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>30</y>
        <width>311</width>
        <height>201</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Isolated Places</string>
      </property>
      <widget class="QPushButton" name="btnAddingNewIsP">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Add new</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnDeleteIsP">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>30</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblAreaID">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>AreaID</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="textAreaID">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>100</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="lblNameOfArea">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Name Of Area</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="textNameOfArea">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>140</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="grpInforOfMan">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>250</y>
        <width>321</width>
        <height>291</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Declaration Of Man</string>
      </property>
      <widget class="QPushButton" name="btnAddingNewDeOfMan">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Add new</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnDeleteDeOfMan">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>30</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="textLocation">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>210</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="textnameOfMan">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>160</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="lblName">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Name Of Man</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lblWhereTolive">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="text">
        <string>Location</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lblManID">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ManID</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="textManID">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>110</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="grpBD">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>511</width>
        <height>181</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Database Section</string>
      </property>
      <widget class="QPushButton" name="btnIsolated">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>110</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Isolated Persons</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnShowing">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>110</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Showing DB</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblNameOfDB">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Enter Dir of DB</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="btnOpenDB">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>171</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Connect DB with Dir</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="textNameOfDB">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>40</y>
         <width>301</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="btnSubmit">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>570</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnExit">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>570</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>200</y>
        <width>421</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QListView" name="listView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>81</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_load">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>550</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>LOAD TABLE</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>888</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
